source/assets/public/index.css

```diff
@@ -308,17 +311,67 @@ body {
     margin-top: 2rem;
   }
 
-.back-to-top {
```

Comment:
@libbyberrie this doesn't seem to be used - can I  check with you that it can be deleted